Add Antigravity configuration instructions (#12730)
flutter/website: src/content/ai/mcp-server.md

Dart and Flutter MCP server with popular tools like Firebase Studio,
Gemini CLI, Gemini Code Assist, Cursor, and GitHub Copilot.

### Antigravity

> To configure the [Antigravity][] to use the Dart and Flutter MCP server,

Reviewer: ```suggestion
To configure [Antigravity][] to use the Dart and Flutter MCP server,
```

use the [MCP Store][]:

1. Locate the **Agent Side Panel** on the right. 

>   (It may be closed; then press Ctrl+Cmd+B to open it).

Reviewer: ```suggestion
  (It might be closed, so press Ctrl+Cmd+B to open it).
```
